english/lego-world/lego-world.md

```diff
@@ -3,4 +3,4 @@ But this world is a Lego version of Willy Wonka and the Chocolate Factory.
 You are actually skiing down the slope of a marshmallow mountain made out of Legos!
 You are surrounded by lollipop trees and gingerbread houses.
 [Everything is awesome!](https://youtu.be/etzMjoH0rJw)
-And you just can't help but bust out some dance moves
\ No newline at end of file
+And you just can't [help](../silence/silencefalls.md) but bust out some dance moves
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.